torch/_higher_order_ops/scan.py

```diff
@@ -1,7 +1,7 @@
 # mypy: allow-untyped-defs
 import functools
 import itertools
-from typing import Callable, List, Tuple
+from typing import Any, Callable, List, Tuple
```

Comment:
For future reference: it sounds like the PR could have been split into "make subgraph inputs be a a slice" and "flatten subgraph output", it's a bit difficult for me to disentangle this

Reply:
okok